Resources/Prototypes/Catalog/Fills/Items/belt.yml

```diff
@@ -148,6 +148,28 @@
       - id: OmnizineChemistryBottle
       - id: MedkitCombatFilled
 
+- type: entity
+  id: ClothingBeltBandolierFilledLethals
+  parent: ClothingBeltBandolier
+  name: bandolier
+  suffix: Lethals
+  components:
+  - type: StorageFill
+    contents:
+      - id: ShellShotgun
+        amount: 60
+
+- type: entity
+  id: ClothingBeltBandolierFilledIncendiary
+  parent: ClothingBeltBandolier
+  name: bandolier
+  suffix: Incendiary
+  components:
+  - type: StorageFill
+    contents:
+      - id: ShellShotgunIncendiary
+        amount: 60
```

Comment:
Don't add 60 entities dog.

Reply:
but that's the capacity of the bandolier

Reply:
Spawning 30-60 entities for 1 bandolier is really bad and means we have to severely limit how many go in game, just make it use ballisticammoprovider.

Reply:
ill have to research that one